### ISSUE TITLE
Toggle for param not on when default value is true

### ISSUE THREAD

**Issue by anthonyshchang** [ViewComponent/lookbook#267]
> **Before opening an issue** please make sure you have consulted the [Lookbook documentation](https://lookbook.build) (in particular the [Troubleshooting section](https://lookbook.build/guide/troubleshooting/)) and have checked the existing issues to see if this has already been reported.

### Describe the bug

When the default value of a `toggle` param is true, the toggle in the UI starts off as toggled off.

### To Reproduce

1. Define a param with the `toggle` type
2. Set the value of that param to `true`
3. Start lookbook and view component

### Expected behavior

If the value of a `toggle` param is `true`, the UI for the param should start of as toggled `on`.

### Version numbers 

Please complete the following information:

* Lookbook: 1.3.3
* ViewComponent: 2.75.0
* Rails: 7.0.4
* Ruby: 3.1.2

**Comment by aduth** [comment]
I just encountered this in our application as well. As a temporary workaround, it worked for us to invert the default to `false`, since Lookbook will always show the toggle as off when initially shown. That may not work for everyone and it would be nice if the default worked as expected.

I noticed that when initially rendered, the toggle input does have the initial value:

```html
<div data-param-input="toggle" x-data="paramsInputComponent({name: 'paramName', value: true})">
  <!-- ... -->
</div>
```

When toggling it on, the markup shows the value as a _string_ of `'true'`:

```html
<div data-param-input="toggle" x-data="paramsInputComponent({name: 'paramName', value: 'true'})">
  <!-- ... -->
</div>
```

Maybe it's the difference of type of boolean and string which is preventing the default from being shown correctly?

**Comment by allmarkedup** [comment]
Hey @anthonyshchang thanks for the bug report and @aduth for chiming in - I have recreated it locally and it is indeed to do with the type casting of values. I think I've actually fixed variations on this bug before but they keep coming back to haunt me :-)

I'll get this fixed now and get a new patch release out as soon as I can. Thanks for your patience 👍

**Comment by allmarkedup** [comment]
Hey all - I've just released v1.4.5 that should fix this bug so I'm going to close this down now. If you are still having any problems after upgrading feel free to reopen :-) Thanks again for bringing it to my attention.
